Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2138,8 +2174,9 @@ void CGContextClearRect(CGContextRef context, CGRect rect) {
     ComPtr<ID2D1CommandList> commandList;
     RETURN_IF_FAILED(deviceContext->CreateCommandList(&commandList));
 
-    deviceContext->BeginDraw();
+    EscapeBeginEndDrawStack();
     deviceContext->SetTarget(commandList.Get());
```

Comment:
I think you're correct. I recall Begin->Set->End being specified in one of the MSDN documents.